packages/package_info_plus/package_info_plus_windows/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0
+- **Breaking change**. Bump minimum dart version to 2.17.0.
```

Comment:
```suggestion

- **Breaking change**. Bump minimum dart version to 2.17.0.
```

Reply:
This is on the wrong CHANGELOG.md according to your previous explanation since this is the windows  CHANGELOG which did get the version bump, so I didn't touch it.

Reply:
> This is on the wrong CHANGELOG.md according to your previous explanation

Didn't get what you mean. The suggestion from me is to add a new empty line between version and text to be consistent in format with rest of changelog file.